src/theme/CodeBlock/CopyButton/index.js

```diff
@@ -9,16 +9,21 @@ export default function CopyButton({ code, className }) {
   const copyTimeout = useRef(undefined);
   const handleCopyCode = useCallback(() => {
     const removeLineNumbers = /^\s*\d+\s/gm;
-    const removeEllipses = /\.{3}/gm;
+    const removeEllipses = /(^\s*\.{3})(?![^a-z])|(^\.{3})/gm;
```

Comment:
@ymekuria What's the first part for? I'd have thought your second part would solve it `(^\.{3})`

Reply:
In testing I found that sometimes the elipses have preceding spaces. The first part ensures that spread operators are not removed when matching elipses that have preceding spaces.

Reply:
cool. thanks for explaining